.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendored_q2view"]
+	path = vendored_q2view
```

Comment:
This path is the branch this is pointing at and will need to be changed when qiime2/q2view#25 is merged